docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  db:
+    image: postgres
```

Comment:
is it possible to make at least the major version explicit? Not that we are planning to use any advanced feature for now, but it could be useful to know

Reply:
Good point, I've added this in